src/components/graph/menus/create-node-menu.js

```diff
@@ -193,26 +215,112 @@ const CreateNodeMenu = ({
             onClose={handleClose}
         >
             {Object.values(NODE_MENU_ITEMS).map((item) => {
+                const isCreate = item.id === 'createNetworkModificationNode';
```

Comment:
This is strnage...
Maybe you should have a NODE_MENU_ITEMS definition containing some nested Menu
ex:
```
const NODE_MENU_ITEMS = {
        MENU_ITEM_A: {
            onRoot: true,
            action: () => foo(),
            id: 'myIdA',
            subMenuItems: {
                MENU_ITEM_B: {
                    id: tutu,
                    ...
                }
            }
        },
```
Then you could create a recursive function, which will render the corresponding JSX Menu with `NestedMenuItem` when needed (i.e when subMenuItems !== undefined and has at least one menu item...)

Reply:
done